lib/msal-node/src/network/HttpClient.ts

```diff
@@ -267,7 +265,6 @@ const networkRequestViaHttps = <T>(
                     // do not destroy the request for the device code flow
                     networkResponse.body["error"] !== Constants.AUTHORIZATION_PENDING) {
                     request.destroy();
```

Comment:
I thought we were deleting this if check?

Reply:
No, this is maintained as the check for the device code flow condition is required, and also since this is at the very end of the logic, it makes sense to maintain this to avoid unintended consequences. 
You can refer to @bgavrilMS 's comment regarding this. Let me know if there are any further questions, thanks